src/library/scala/collection/ArrayOps.scala

```diff
@@ -183,47 +183,47 @@ object ArrayOps {
   *
   *  @tparam A   type of the elements contained in this array.
   */
-final class ArrayOps[A](val xs: Array[A]) extends AnyVal {
+final class ArrayOps[A](val as: Array[A]) extends AnyVal {
```

Comment:
This should probably be made private[this] too right?

Reply:
Just `private` will do.

Reply:
You can keep the `xs` name and just refer to it as `this.xs` where necessary